scripts/ricardeos/README.md

```diff
@@ -0,0 +1,20 @@
+# Purpose
+The `recardeos.py` imports or exports recardian contracts to and from a contracts abi
```

Comment:
This file has a lot `recardeos` and `recardian`, where they should be `ricardeos` and `ricardian`